renderers/Context.js

```diff
@@ -129,14 +130,17 @@ Context.prototype.draw = function draw() {
 };
 
 /**
- * Gets the size of the parent element of the DOMRenderer for this context.
+ * Determines the root element's offset size.
  *
  * @method
  *
  * @return {undefined} undefined
  */
 Context.prototype.getRootSize = function getRootSize() {
-    return this.DOMRenderer.getSize();
```

Comment:
@DnMllr Are you ok with this? I think it's more intuitive to fetch the size from the actual context then the DOMRenderer. Sorry, didn't mean to put this in this PR.